### PR TITLE
fix: group members card

### PR DESCRIPTION
**Pull request by ana-yldio**
https://trello.com/c/rV11LRwM/1778-labs-are-showing-on-group-pages-members-component

### CODE REVIEW

**Review by niezgoda — APPROVED** [review]

**Review by niezgoda — APPROVED** [review]

File: packages/react-components/src/molecules/MembersList.tsx [hunk 1]
```diff
@@ -109,12 +110,13 @@ const MembersList: React.FC<MembersListProps> = ({
                 addToColumnStyles,
                 singleColumn || multiColumnAddToColumnStyles,
                 textStyles,
-                labStyles,
+                labStyles(!labsList.length),
```
Comment (kmaid) [inline packages/react-components/src/molecules/MembersList.tsx:113]:
```suggestion
              !!labsList.length && labStyles,
```

File: packages/react-components/src/molecules/MembersList.tsx [hunk 2]
```diff
@@ -46,9 +46,10 @@ const textStyles = css({
   color: lead.rgb,
 });
 
-const labStyles = css({
-  padding: `0 0 ${24 / perRem}em`,
-});
+const labStyles = (hide = true) =>
+  css({
+    padding: hide ? 0 : `0 0 ${24 / perRem}em`,
+  });
```
Comment (kmaid) [inline packages/react-components/src/molecules/MembersList.tsx:52]:
```suggestion
const labStyles = css({
    padding: `0 0 ${24 / perRem}em`,
  });
```

File: packages/react-components/src/molecules/MembersList.tsx (hunk 1 above)
Comment (kmaid) [reply packages/react-components/src/molecules/MembersList.tsx:113]: This should be more readable

File: packages/react-components/src/organisms/GroupMembersSection.tsx [hunk 3]
```diff
@@ -26,7 +26,12 @@ const GroupMembersSection: React.FC<GroupMembersSectionProps> = ({
       <MembersList
         singleColumn
         members={leaders.map(({ user, role }) => ({
-          ...user,
+          id: user.id,
+          displayName: user.displayName,
+          firstName: user.firstName,
+          lastName: user.lastName,
+          avatarUrl: user.avatarUrl,
+          teams: user.teams,
```
Comment (kmaid) [inline packages/react-components/src/organisms/GroupMembersSection.tsx:34]:
```suggestion
          members={leaders.map(({ user: { labs, ...user}, role }) => ({
          			...user,
          			role
```

**Review by kmaid — CHANGES_REQUESTED** [review]

**Pull request closed without merging**